db/migrate/20200915080831_create_data_memories.rb

```diff
@@ -0,0 +1,9 @@
+class CreateDataMemories < ActiveRecord::Migration[6.0]
+  def change
+    create_table :data_memories do |t|
+      t.integer :name
```

Comment:
Looks like the ERD design does not match very well with code? For example, there is a `size` attribute in your ERD, but here you created `name`.

Reply:
I think a data memory entity should contain both `name` and `size`?

Reply:
I have changed the name column to size. Thanks